packages/components/src/components/ResizableBox/ResizableBox.tsx

```diff
@@ -19,6 +19,7 @@ export type ResizableBoxProps = {
     updateWidthOnWindowResize?: boolean;
     updateHeightOnWindowResize?: boolean;
     zIndex?: ZIndexValues;
+    onSaveWidth?: (width: number) => void;
```

Comment:
Could you please provide more context about the suggestion? I'd avoid using "end" in this prop naming.

Reply:
Makes totally sense in case we want to let the component accept some function that is gonna simply be executed on mouse up, no matter what it is. But I think this is more specific case, added for specific purpose, and the function accepting specific prop(s). Hence we should be more specific also in naming.. if we don't decide we want to be super universal here. I'm not sure yet as both approaches make sense and have both pros and cons.

Reply:
I don't agree that this is the special use case. It's `onChange` function that is called once in the end of the click, nothing else. You don't have any special logic related to saving. What name would you suggest?

Reply:
It is specific because it works with `newWidth` actually (or will work with the box dimensions once I push the update for the height). The name should reflect its use case, shouldn't be totally universal because there's so much stuff we could do with the box on mouse up, for example to adjust color, highlighting, focus etc. 

Not sure about the exact name yet. But using "end" isn't the best as it can also make one feel it is gonna be something related to some specific part of the box, like it's for css when "flex-end" value is used, for example. So I'd be careful to prevent potential confusions, maybe "onResizeFinish" would be good enough. Even better would be if it reflected that it has something to do with the dimensions of the box (and not color or z-index etc etc). The more specific we are, the more control we have over the behavior, the better usability of the component, less bugs etc.

Reply:
Please don't overthink stuff. I don't want to spend so much time on discussing naming all the time. If you think I'm wrong try to ask someone else in the team.

I still think `onResizeEnd` is valid here. In arguments, you are reflecting the state of the component in the moment when you ended the touch. If there will be more values than width and height, we will create an object for it. 
Also, `End` suffix is widely used across events like this. For example in official API for touch events: https://developer.mozilla.org/en-US/docs/Web/API/Element/touchend_event

Reply:
> Please don't overthink stuff. I don't want to spend so much time on discussing naming all the time. If you think I'm wrong try to ask someone else in the team.

Thank you for sharing your opinion. Without discussion or having different opinion, I'd learn nothing. I really appreciate you invest time in the discussion and provide ideas, thoughts, information.

> I still think onResizeEnd is valid here. In arguments, you are reflecting the state of the component in the moment when you ended the touch. If there will be more values than width and height, we will create an object for it.
Also, End suffix is widely used across events like this. For example in official API for touch events: https://developer.mozilla.org/en-US/docs/Web/API/Element/touchend_event

From this point of view, 'onResizeEnd' makes more sense to me now. And it's great to inspire there when naming functions. On the other hand, we're still adding slightly different thing to ResizableBox here, we're trying to get some data to save it in the Redux store. My original naming isn't the best but I'd rather not to mix Redux stuff with anything else. 'onResizeEnd' seems also to me too general comparing to that, general naming for a very specific use case, as it won't work for anything else than saving the width (and/or height) - this can be misleading. So I'm not sure..

To sum up, yes, naming can be hard and also a very subjective topic. Work in progress. And that's why it's good to discuss. But let's step away from it for a while to prevent spending "so much time on discussing naming all the time" and let's see if anything better comes to our minds ;)

Reply:
> ...The name should reflect its use case...

In general I found very useful to name things base on *what* they are, rather then *how* are they used. Component is placed in the separate context (package) of abstraction and has its API which I feel shall be rather generic.

Reply:
Both `onResizeFinish` and `onResizeEnd` are totally fine for me.